Bio/_py3k/__init__.py

```diff
@@ -94,11 +94,16 @@ def _is_int_or_long(i):
         return isinstance(i, int)
 
     import io
+    import locale
 
     # Python 3.4 onwards, the standard library wrappers should work:
     def _binary_to_string_handle(handle):
         """Treat a binary (bytes) handle like a text (unicode) handle."""
-        wrapped = io.TextIOWrapper(io.BufferedReader(handle))
+        try:
+            encoding = handle.headers.get_content_charset()
```

Comment:
I would add a comment here that you are checking the HTML headers which would be present if this was a network handle. Without context I find this is hard to understand.